app/views/2017/articles/_list.html.erb

```diff
@@ -1,5 +1,7 @@
+<% skip_preferred_localization ||= skip_preferred_localization %>
```

Comment:
is this an erb thing where you need to save to a variable to use it?

Reply:
It's the pattern I've seen if you want to have a default. 

Ie, it's only set in this exceptional case. All other cases, no local is set or needed.